examples/decoding/plot_decoding_xdawn_transformer.py

```diff
@@ -40,9 +40,10 @@
                 add_eeg_ref=False, verbose=False)
 
 X = epochs.get_data()
-y = label_binarize(epochs.events[:, 2], classes=[1, 3])
+y = label_binarize(epochs.events[:, 2], classes=[1, 3]).ravel()
 
 clf = make_pipeline(XdawnTransformer(n_components=3),
+                    Vectorizer(),
```

Comment:
Travis says this isn't imported anywhere:

https://travis-ci.org/mne-tools/mne-python/jobs/146206895#L2969

Does this run locally?